src/Uno.UI/UI/Xaml/UIElement.EventRegistration.wasm.cs

```diff
@@ -113,18 +106,23 @@ public bool Dispatch(EventArgs eventArgs, string nativeEventPayload)
 						args = _payloadConverter(_owner, nativeEventPayload);
 					}
 
-					if (args is RoutedEventArgs routedArgs)
-					{
-						routedArgs.CanBubbleNatively = _canBubbleNatively;
```

Comment:
Is `CanBubbleNatively` still set elsewhere?

Reply:
Yes you only have to set it when you create the `RoutedEventArgs` (the property's getter and setter are internal, and users cannot create a `RoutedEvent`), which in my point of view is much more discoverable.

I did this to reduce the complexity of `EventRegistration` which had (has) way to much hard coded specific cases. Anyway I think that routed events should not use the `EventRegistration` anymore as it implies serialization/parsing and assumes that you may have multiple listeners for a same event (which is not the case for pointer for instance). Actually that's what I did in my branch which fixes https://github.com/unoplatform/uno/issues/2116, and once I've merge it, the last routed event which will still use the `EventRegistration` will be the keyboard events.

Reply:
FYI, I moved it for the keyboard here https://github.com/unoplatform/uno/pull/3003/files#diff-6f3c127d07fda497219e71a7d9b763a1R673 Pointers events was already set it properly in the args ctor, depending of few parameters.